lib/api/src/sys/store.rs

```diff
@@ -309,6 +326,17 @@ impl<'a> StoreMut<'a> {
     pub(crate) unsafe fn from_raw(raw: *mut StoreInner) -> Self {
         Self { inner: &mut *raw }
     }
+
+    /// Sets the unwind callback which will be invoked when the call finishes
+    pub fn on_called<F>(&mut self, callback: F)
```

Comment:
Can we add a new comment linking to the asyncify refactor afterwards:

```
// TODO: issue #XYZ 
```

Reply:
done